src/features/dsComponents/body/BodyHeader.tsx

```diff
@@ -38,9 +39,9 @@ const BodyHeader: React.FC<BodyHeaderProps> = ({
           TODO (boandriy): Once the preview row count is filtered by fetch pagination -
           use global filtering variable instead of hardcoded 100
         */}
-        <div className={'mr-4'}>
-          Previewing 100 of {numeral(structure?.entries).format('0,0')} rows
-        </div>
+        {+entries > 100 && <div className={'mr-4'}>
```

Comment:
from some in-browser experimenting, it doesn't look like the `+` unary operator works well with string representations of numbers that include commas:

```
var s = "100,000"
+s // NaN
+s > 100 // false
```

Since we specifically get the `entries` variable from using `format('0,0')`, there is a good chance the `entries` string will have a "comma"

Try using `numeral(entries)` rather than `+`